### DISCUSSION TITLE
Trouble installing and running v9 on PHP (documentation out of date?)

### DISCUSSION THREAD

**Discussion by iamkeir** [directus/directus#3220] [Archive]
Hi folks,

I am trying to install Directus but encountering a series of problems, and the documentation looks like it may be lacking/out of date.

If I use the new `npx create-directus-project my-project`, after it is installed, to run it with PHP, I am unsure where I should be pointing my virtual host as this installation approach is not listed here: https://docs.directus.io/getting-started/installation.html).

If I am use the [Git instructions](https://docs.directus.io/installation/git.html), the documentation says to point the virtual host at `/directus/public` but there is no public folder here, instead it is found in `/directus/app/public`. If I point my VH at that public folder instead, when I visit the URL, it shows a blank page with "Loading..." as the browser title but nothing ever loads.

I am running MAMP Pro with PHP v7.3.1 and MySQL 5.7.25, with all relevant extensions/modules enabled, and permissions set correctly.

From reading around as much as possible, it kinda looks like v9 is currently Node only, and the PHP version is not yet available - is that the case?

Any guidance would be greatly appreciated, thank you.

**Comment by rijkvanzanten** [comment]
Heya! You are correct. The docs on docs.directus.io are (at the time of writing) still for v8.. We're currently overhauling the docs for v9 (see https://github.com/directus/directus/tree/main/docs), which are temporarily deployed to next.docs.directus.io. We hope to have the new docs live somewhere early next week 👍

**Reply by iamkeir** [reply]
Brill, thanks - yep had someone confirm on Discord too. Will post the extra info for usefulness as well.

**Comment by iamkeir** [comment]
Confirmed in Discord as well - PHP version not ready yet (80% done, hopefully ready next few months, but maybe community-maintained).

In the meantime, for anyone else having these difficulties:
1) Run `npx create-directus-project my-project`
2) Follow instructions presented
3) View site at http://localhost:8055

**Comment by surfmuggle** [comment]
Out of curiosity what where the pros and cons to focus on Node.js first?
I assume an article / post on for example reddit could boost traffic to directus.

**Reply by benhaynes** [reply]
Hopefully this helps... 👍 
https://directus.io/articles/introducing-directus-9/

**Reply by surfmuggle** [reply]
The link above no longer works. This is likely the new link: https://directus.io/blog/introducing-directus-9/
